lib/modules/datasource/crate/index.ts

```diff
@@ -371,4 +377,39 @@ export class CrateDatasource extends Datasource {
 
     return [packageName.slice(0, 2), packageName.slice(2, 4), packageName];
   }
+
+  private static releaseTimestampSchema = z
+    .object({
+      version: z.object({
+        created_at: z.string(),
+      }),
+    })
+    .transform(({ version: { created_at } }) => created_at);
```

Comment:
Don't we normally put these in a schema.ts?

Reply:
Yes, didn't feel like it creating the new file, will do this later

Reply:
at least move it out of the class